test/specs/wrapper.spec.js

```diff
@@ -15,4 +16,37 @@ describeWithShallowAndMount('Wrapper', mountingMethod => {
         .with.property('message', message)
     })
   })
+
+  describe('enableAutoDestroy', () => {
```

Comment:
We somehow need to reset state here: https://circleci.com/gh/vuejs/vue-test-utils/13058?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Not quite sure how yet... :thinking:

Reply:
I suspect the state inside the module is kept between the tests? This is normal as modules once imported, are used from the cache.

I know how to do this with Jest, however not sure with Mocha...

You could also export an internal method to clear it? That is how we did it with Vuelidate 1 before, for a similar case.

Reply:
Maybe we want a `disableAutoDestroy`?

Reply:
I dont see why someone would use it, but when there is a way to enable, should be a way to disable :)

Reply:
> I dont see why someone would use it

Yes, I agree. It was mainly meant for testing but I found a different solution now. :muscle:

Reply:
Wouldnt exporting a `reset` method, that sets the boolean to false have been enough? :D Object.assign-ing like that the state is clever, but also bending the rules hehe

Reply:
@dobromir-hristov I don't mind a `reset` either. :smiley: 

Do you see why my

```
import { autoDestroyState } from '~vue/test-utils/src/auto-destroy'
```

doesn't work though? (https://circleci.com/gh/vuejs/vue-test-utils/13062?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

Do I have to publicly expose the state / reset function?

Reply:
I think you should use relative paths?

Reply:
ah, that's not very pretty. :grimacing: but :man_shrugging:

Reply:
That doesn't seem to work either. :thinking: 

https://circleci.com/gh/vuejs/vue-test-utils/13065?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Does it work on your local?

Reply:
nope :disappointed: